l10n_it_fatturapa_out/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 
 {
     'name': 'ITA - Fattura elettronica - Emissione',
-    'version': '12.0.1.2.1',
+    'version': '12.0.1.5.0',
```

Comment:
c'è un motivo particolare per cui hai apportato questo cambio di versione? In caso contrario conviene non modificarla e lasciarlo fare a`ocabot`.

Reply:
Grazie della review! No direi che era una modifica inclusa nel cherry-pick, l'ho rimossa